ee/server/service/maintained_apps.go

```diff
@@ -26,19 +27,19 @@ func (svc *Service) AddFleetMaintainedApp(
 	appID uint,
 	installScript, preInstallQuery, postInstallScript, uninstallScript string,
 	selfService bool,
-) error {
+) (uint, error) {
```

Comment:
Nit: might be worth naming the return values just to make it clear what the `uint` is (it's not obvious by the name of the method, had to scroll to the end to see it was title ID).